content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -100,13 +100,35 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
 ### Install using native package management
 
 {{< tabs name="kubectl_install" >}}
-{{< tab name="Ubuntu, Debian or HypriotOS" codelang="bash" >}}
-sudo apt-get update && sudo apt-get install -y apt-transport-https gnupg2 curl
-curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list
-sudo apt-get update
-sudo apt-get install -y kubectl
-{{< /tab >}}
+{{% tab name="Ubuntu, Debian or HypriotOS" %}}
+
+1. Update the `apt` package index and install packages needed to use the Kubernetes `apt` repository:
+
+   ```shell
+   sudo apt-get update
+   sudo apt-get install -y apt-transport-https ca-certificates wget
+   ```
+
+2. Download the Google Cloud public signing key:
+
+   ```shell
+   sudo wget -O /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+   ```
+
```

Comment:
How about package pinning? I'm thinking of https://wiki.debian.org/DebianRepository/UseThirdParty#Complete_example

Perhaps that's something for a follow-up issue?